Assignable.html

```diff
@@ -1,109 +1,19 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Don't worry about turning the existing HTML files into redirects, it's fine to remove them entirely. People would be linking to a specific Boost version documentation that would have this file, so it's fine for the new version to not have it.

Reply:
> People would be linking to a specific Boost version documentation that would have this file, so it's fine for the new version to not have it.

Not necessarily. I've been using version-neutral links in some Boost docs. Though I'm not sure I used links to Boost.Utility.

Reply:
The html files in the root directory are so old that I assumed that's the only reason they are still there.
On the other hand, they were mostly documenting stuff that's very old and not even part of boost.utility anymore.
I tried to be conservative on that but I doubt many people rely on these links anyway.

Reply:
> The redirect files pointing to facilities moved to other libraries should be retained

Done: d37d863f9b646496553eae5a8e9318c944761a96

> If facilities no longer exist, (...) those can be removed.

Broken links don't redirect to Utility's `index.html` though: https://www.boost.org/doc/libs/1_77_0/libs/utility/broken_link.html
They look rather ugly. Is it safe to assume there's consensus on that yet?

Reply:
If someone is linking to `boost.org/libs/utility/something.html` and `something` as a Boost facility doesn't even exist anymore, then there's no value in that link going to `utility/index.html` so it's fine if it's broken.